### PR TITLE
Epub renderer (WIP)

### PR DESCRIPTION
**Pull request by christianmemije**
### Summary

Testing channel: 
9096a64765c64af18519a9d9509e64a9

Opened issues for remaining issues.

### Reviewer guidance

…

### References

----

### Contributor Checklist

- [ ] Contributor has fully tested the PR manually
- [ ] PR has the correct target branch and milestone
- [ ] PR has 'needs review' or 'work-in-progress' label
- [ ] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] If there are any front-end changes, before/after screenshots are included
- [ ] If this is an important user-facing change, PR or related issue has a 'changelog' label

### Reviewer Checklist

- [ ] Automated test coverage is satisfactory
- [ ] Reviewer has fully tested the PR manually
- [ ] PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- [ ] External dependencies files were updated (`yarn` and `pip`)
- [ ] Documentation is updated
- [ ] Link to diff of internal dependency change is included
- [ ] CHANGELOG.rst is updated for high-level changes
- [ ] Contributor is in AUTHORS.rst

### CODE REVIEW

**Comment by indirectlylit** [comment]
@christianmemije any idea why the tests are failing here?

**Review by indirectlylit — APPROVED** [review]
great job!

File: kolibri/plugins/document_epub_render/assets/tests/BottomBar.spec.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    expect(wrapper.exists()).toBe(true);
+  });
+  it('should not display a heading if none is provided', () => {
+    const wrapper = createWrapper();
+    expect(wrapper.contains('h3')).toBe(false);
+  });
+  it('should display a heading if one is provided', () => {
+    const heading = 'Chapter 1';
+    const wrapper = createWrapper({ heading });
+    expect(wrapper.contains('h3')).toBe(true);
```
Comment (jonboiser) [inline kolibri/plugins/document_epub_render/assets/tests/BottomBar.spec.js:32]: A stronger test might be to assert that the h3 has the correct text

```
expect(wrapper.find('.header').text()).toEqual('Chapter 1')
```

File: kolibri/plugins/document_epub_render/assets/tests/SearchSideBar.spec.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+  it('should allow parent to focus on input box', () => {
+    const wrapper = createWrapper();
+    wrapper.vm.focusOnInput();
+    const elementThatIsFocused = document.activeElement;
+    expect(elementThatIsFocused.classList.contains('search-input')).toBe(true);
+  });
+
+  it('should highlight search terms', () => {
+    const wrapper = createWrapper();
+    wrapper.vm.searchQuery = 'biology';
```
Comment (jonboiser) [inline kolibri/plugins/document_epub_render/assets/tests/SearchSideBar.spec.js:28]: You Could use `wrapper.setData()` to change these values as well

**Pull request merged**